app/src/main/java/com/example/next_show/fragments/ShowDetailFragment.java

```diff
@@ -128,25 +128,28 @@ public void onClick(View v) {
                 currShow.setUserLiked("liked");
                 Toast.makeText(getActivity(), "You liked this show!", Toast.LENGTH_SHORT).show();
                 if(alreadySaved){
```

Comment:
This if block is not empty.  Should the TODO still be there?

Reply:
Yes! This is what I hope to fix this week and have the onClick trigger a saveInBackground for the database for the updated rating